plugins/leemons-plugin-academic-calendar/frontend/src/hooks/keys/listRegionalConfigs.js

```diff
@@ -0,0 +1,13 @@
+export const allListRegionalConfigsKeys = [
+  {
+    plugin: 'plugin.academic-calendar',
+    scope: 'list-regional-configs',
```

Comment:
```suggestion
    action: 'list',
    scope: 'regional-configs',
```

Reply:
Done.

Reply:
Not how I indicated :D, I fix it, no problem 😉